libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -352,6 +352,7 @@ void AC_AutoTune_Multi::load_test_gains()
         if (axis == YAW_D) {
             attitude_control->get_rate_yaw_pid().kD(tune_yaw_rd);
         } else {
+            attitude_control->get_rate_yaw_pid().kD(0.0);
```

Comment:
```suggestion
            attitude_control->get_rate_yaw_pid().kD(0.0f);
```

Reply:
I think we are moving away from adding the f at the end of numbers now.

Reply:
Yes, no need for the `f`.